src/command.rs

```diff
@@ -207,17 +207,19 @@ where
 ///   [`ssh(1)`]: https://linux.die.net/man/1/ssh
 ///   [`env(1)`]: https://linux.die.net/man/1/env
 #[derive(Debug)]
-pub struct Command<'s> {
-    session: &'s Session,
+pub struct OwnedCommand<S> {
```

Comment:
Can we rename this to `CommandAnyClonable`?

Reply:
Ah sorry I had not seen your comment before doing a bit more shuffling around, squashing the commits, and writing docs.

I ended up renaming the internal command back to `Command`, and doing the aliasing on the edge in `lib.rs`, with the inner generic `Command` being exported as `OwningCommand` (seems better than `Owned`), and the `openssh::RemoteChild` and `openssh::Command` aliases being created there (`lib.rs`) rather than inside the crate guts.

I can still rename `crate::command::Command` if you prefer some other name tho.